src/app/features/event/event-survey/event-survey-single/event-survey-single.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="surveys">
+  <form [formGroup]="surveyForm" class="surveys__single">
+    <h3 class="surveys__single__header font-heading-3">{{ survey.question }}</h3>
+    <ng-container *ngIf="survey.isMultipleChoice">
+      <mat-checkbox
+        *ngFor="let answer of survey.answers"
+        class="surveys__single"
+        formControlName="answer"
+        [value]="answer.value">
+        <p class="font-text-1">{{ answer.value }}</p>
+      </mat-checkbox>
+    </ng-container>
+    <ng-container *ngIf="!survey.isMultipleChoice">
+      <mat-radio-group formControlName="answer">
+        <mat-radio-button *ngFor="let answer of survey.answers" class="surveys__single" [value]="answer">
+          <p class="font-text-1">{{ answer.value }}</p>
+        </mat-radio-button>
+      </mat-radio-group>
+    </ng-container>
```

Comment:
Zastanawiam się czy zamiast `<ng-container *ngIf="!survey.isMultipleChoice">` może zastosować `else` w pierwszym `ngIf` ? Nie jestem pewny czy to jakaś wielka różnica, tylko taka luźna myśl.

Reply:
Wolałem podzielić na 2 ify ze względu na czytelność kodu. Explicite zaznaczyć, że to ma się wywołać na `isMultipleChoice=false`

Reply:
ze względu na performance nie ma w sumie różnicy, pytanie jeżeli chodzi o spójność projektową :)